src/main/java/Duke.java

```diff
@@ -6,5 +9,73 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        Scanner sc = new Scanner(System.in);
+        String hello = "Hello! I'm Duke";
+        String question = "What can I do for you?";
+        System.out.println(hello + "\n" + question);
+        String ans = sc.nextLine();
+        ArrayList<Task> listOfStuff = new ArrayList<>();
```

Comment:
Maybe not a good idea to use "listOfStuff" as a name since it's too generic, perhaps can use "tasks" instead?